codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/AWSHTTPProtocolCustomizations.kt

```diff
@@ -65,7 +65,10 @@ abstract class AWSHTTPProtocolCustomizations : DefaultHTTPProtocolCustomizations
             // Internal service clients are contained in aws-sdk-swift targets that ARE NOT vended externally
```

Comment:
Changes in this file follow same logic as the one in STS web identity. Essentially adding internal client instantiatino & saving it to auth options' identity properties, but now for SSO and SSO OIDC.